go/fory/fory.go

```diff
@@ -174,8 +174,19 @@ type Fory struct {
 	metaContext    *MetaContext
 }
 
-func (f *Fory) RegisterTagType(tag string, v interface{}) error {
-	return f.typeResolver.RegisterTypeTag(reflect.ValueOf(v), tag)
+func (f *Fory) RegisterType(
+	v interface{},
+	namespace string,
+	typeName string,
+) error {
+	return f.typeResolver.RegisterNamedType(reflect.TypeOf(v), namespace, typeName)
+}
+
+func (f *Fory) Register(
```

Comment:
Do we have a API that register by id?

Reply:
Maybe register API should be register by ID, and we have a func `RegisterNamedType` which register by name